src/lib/statistics-service/recorder.ts

```diff
@@ -57,7 +52,7 @@ export async function sendEventToRecorder(
   const abtesting = ABTestingService.getValues()
   const benefitsStats: StatisticsRecord[] = []
   const totalResults = benefits.length
-  const eventHashId = identifyEvent(matomo)
+  const eventHashId = identifyEvent()
```

Comment:
Suite du commentaire au dessus
```suggestion
  const eventHashId = await identifyEvent()
```

Reply:
Pareil j'ai enlevé les async (il n'y avait aucun `return` pour que ça fonctionne même si c'était réélement async